ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -99,17 +107,25 @@ const TopologyComponent = () => {
         };
     }, [controller]);
 
+    const selectedIds = selectedObj ? [selectedObj.id] : [];
+
     return (
         <TopologyView
             sideBar={
-                <TopologySideBar show={!!selectedId} resizable onClose={() => setSelectedId(null)}>
-                    <div style={{ height: '100%' }}>{selectedId}</div>
+                <TopologySideBar
+                    show={!!selectedObj}
+                    resizable
+                    onClose={() => setSelectedObj(null)}
+                >
+                    {selectedObj?.type === 'group' && <div>Group</div>}
+                    {selectedObj?.type === 'node' && <DeploymentSideBar />}
+                    {selectedObj?.type === 'edge' && <div>Edge</div>}
```

Comment:
i don't think we're going to have a separate side panel for edges from what i recall. @alanonthegit correct me if i'm wrong

Reply:
Yea true. Let me remove that